src/interaction/command/recache.ts

```diff
@@ -1,9 +1,10 @@
-import { schemaTypes, SchemaTypes } from "./../../util/manager/database";
 import { CIRCLE_PERMS } from "./../../util/perms";
 import SlashCommand, {
   SlashCommandContext,
 } from "../../api/interaction/slashcommand";
+import { schemaTypes } from "../../util/manager/firestore";
 
+const schemas = schemaTypes.options;
 export default class RecacheCommand extends SlashCommand {
   public constructor() {
     super({
```

Comment:
Now that I have perms to see roles I can create different variables for roles, this was originally temporary because I didn't know all roles necessary